beam/portable_beam.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+readonly BEAM_VERSION="2.6.0-nightly"
+readonly LOCAL_JAR_NAME='beam-runners-flink_2.11-job-server.jar'
+# TODO: find a way to automatically get latest nightly snapshot
+readonly NIGHTLY_SNAPSHOT_PATH="beam-runners-flink_2.11-job-server/2.6.0-SNAPSHOT/beam-runners-flink_2.11-job-server-2.6.0-20180719.181838-16.jar"
+readonly NIGHTLY_SNAPSHOT_REPO="https://repository.apache.org/content/repositories/snapshots/org/apache/beam"
+readonly RELEASE_SNAPSHOT_REPO="http://repo1.maven.org/maven2/org/apache/beam"
+# TODO: add 2.6.0 release snapshot path when it becomes available
+readonly RELEASE_SNAPSHOT_PATH=""
+readonly SERVICE_INSTALL_DIR='/usr/lib/beam-job-service'
+readonly SERVICE_WORKING_DIR='/var/lib/beam-job-service'
+readonly SERVICE_WORKING_USER='yarn'
+
+readonly DOCKER_FLINK_SUBNET="${DOCKER_FLINK_SUBNET:-192.168.33.0/24}"
+readonly DOCKER_FLINK_GATEWAY="${DOCKER_FLINK_GATEWAY:-192.168.33.1}"
+readonly DOCKER_FLINK_DEVICE="${DOCKER_FLINK_DEVICE:-docker-flink0}"
+readonly DOCKER_FLINK_NETWORK_NAME="${DOCKER_FLINK_NETWORK_NAME:-flink}"
+
+function is_master() {
+  local role="$(/usr/share/google/get_metadata_value attributes/dataproc-role)"
+  if [[ "$role" == 'Master' ]] ; then
+    true
+  else
+    false
+  fi
+}
+
+function snapshot_url() {
+  if [[ "${BEAM_VERSION}" == '2.6.0-nightly' ]] ; then
+    echo "${NIGHTLY_SNAPSHOT_REPO}/${NIGHTLY_SNAPSHOT_PATH}"
+  else
+    echo "Only 2.6.0 nightly snapshots are currently supported" >&2
+    exit 1
+  fi
+}
+
+function install_job_service() {
+  mkdir -p "${SERVICE_INSTALL_DIR}"
+  pushd "${SERVICE_INSTALL_DIR}"
+  curl -o "${LOCAL_JAR_NAME}" "$(snapshot_url)"
+  popd
+  mkdir -p "${SERVICE_WORKING_DIR}"
+  chown -R "${SERVICE_WORKING_USER}" "${SERVICE_WORKING_DIR}"
+  cat > "/etc/systemd/system/beam-job-service.service" <<EOF
+[Unit]
+Description=Beam Job Service
+After=default.target
+
+[Service]
+Type=simple
+User=${SERVICE_WORKING_USER}
+WorkingDirectory=${SERVICE_WORKING_DIR}
+ExecStart=/usr/bin/java -jar ${SERVICE_INSTALL_DIR}/${LOCAL_JAR_NAME} --job-host=localhost:8099
+Restart=on-failure
+
+[Install]
+WantedBy=multi-user.target
+EOF
+systemctl enable beam-job-service
+}
+
+function run_job_service() {
+  systemctl restart beam-job-service
+}
+
+function create_flink_network() {
+  if docker network list --format '{{ .Name }}' | grep -q "^${DOCKER_FLINK_NETWORK_NAME}$"; then
+      echo "Docker network ${DOCKER_FLINK_NETWORK_NAME} already exists."
+  else
+    docker network create \
```

Comment:
What's the purpose of this bridge network? The Flink runner only uses/supports host network mode as of now anyway.

Reply:
This must be out of date then.  It originally came from the `docker_flink_network.sh` script, which I thought we were still running.  I'll remove it.

Reply:
Yes, we started under the assumption that we would use a special network for this, but switched to using host networking (at least for now), so this is no longer needed.

Reply:
removed.